### DISCUSSION TITLE
BUTTON_LONG_PRESS_HOLD time configurable?

### DISCUSSION THREAD

**Discussion by gknops** [espressif/esp-iot-solution#390] [Q&A] [answered]
I did not find this answered in the documentation: when long-pressing a button the sequence is
- BUTTON_LONG_PRESS_START
- BUTTON_LONG_PRESS_HOLD (rapidly repeated)
- BUTTON_LONG_PRESS_UP

Is there a way to configure the repeat rate of the `BUTTON_LONG_PRESS_HOLD` calls? Right now they seem to be called very rapidly, I'd like to be able to configure that time, say 200ms (meaning while the button is held the callback is called every 200ms).

**Comment by lijunru-hub** [comment, marked as answer]
You chan change this config: BUTTON_SERIAL_TIME_MS to 200ms.

menuconfig -> IOT Button -> BUTTON_SERIAL_TIME_MS

**Reply by gknops** [reply]
This works perfectly, thank you!
